README.md

```diff
@@ -1,19 +1,49 @@
-Absolute
-=============
-
+# Absolute - Web based O2O service platform
 [![Linux/Max Build Status](https://travis-ci.org/lunchclass/absolute.svg?branch=master)](https://travis-ci.org/lunchclass/absolute/branches)
 [![Windows Build status](https://ci.appveyor.com/api/projects/status/099u4iekeny4lpsa/branch/master?svg=true)](https://ci.appveyor.com/project/romandev/absolute/branch/master)
 [![Coverage Status](https://coveralls.io/repos/github/romandev/absolute/badge.svg?branch=master)](https://coveralls.io/github/romandev/absolute?branch=master)
 
-### Welcome to Absolute!
+## Welcome to Absolute!
 
 Absolute is a web-based order platform like siren-order for Starbucks.
 We are focused on creating the platform that is portable, cheaper, easier,
 decentralized and appless.
 
-## Documentation
+**Easier:**  No 'App' install but you can use service. You don't need create account and login any more
```

Comment:
- Easier
  - No 'App' install...
- Cheaper
  - Absolute is cheaper...
- Portable
  - Asbolute support multiple...